gems/pending/appliance_console/cli.rb

```diff
@@ -48,13 +52,25 @@ def database?
     end
 
     def local_database?
-      hostname && local?(hostname)
+      database? && local?(hostname)
     end
 
     def certs?
       options[:postgres_client_cert] || options[:postgres_server_cert] || options[:http_cert]
     end
 
+    def uninstall_ipa?
+      options[:uninstall_ipa]
```

Comment:
ok, I'm fine with this as is, but I'm also thinking...

this might be nicer if the set of options that are "actions" are declared as an Array somewhere earlier.  Then the educate check can be more around what was given on the command line without worrying about the value.  Something like

```ruby
Trollop.educate unless ACTIONS.any? { |a| options.send("#{a}_given") }
```